### PR TITLE
[new release] imagelib (20200929)

### PR DESCRIPTION
**Pull request by mseri**
Library implementing parsing of image formats such as PNG, BMP, PPM

- Project page: <a href="https://github.com/rlepigre/ocaml-imagelib">https://github.com/rlepigre/ocaml-imagelib</a>
- Documentation: <a href="https://rlepigre.github.io/ocaml-imagelib">https://rlepigre.github.io/ocaml-imagelib</a>

##### CHANGES:

- Port to dune 2.0

### CODE REVIEW

File: packages/imagelib/imagelib.20200929/opam [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+   the 'imagelib-unix' OPAM package. See that package description for more info.
+
+As imagelib only requires 'decompress', it can be used together with
+js_of_ocaml to compile projects to Javascript, or from MirageOS unikernels.
+"""
+opam-version: "2.0"
+maintainer: "rodolphe.lepigre@inria.fr"
+bug-reports: "https://github.com/rlepigre/imagelib/issues"
+homepage: "https://github.com/rlepigre/imagelib"
+dev-repo: "git+https://github.com/rlepigre/imagelib.git"
```
Comment (kit-ty-kate) [inline packages/imagelib/imagelib.20200929/opam:27]: These links are broken. The `ocaml-` prefix is missing it seems.

Comment (mseri) [reply packages/imagelib/imagelib.20200929/opam:27]: Thanks. Well spotted. I will fix them here and separately do it on the image lib repo

**Comment by kit-ty-kate** [comment]
could you return the fix upstream? There was a missing `@install` to give to dune. Otherwise dune would not install anything when `opam install --with-test` is called.

**Comment by mseri** [comment]
Geez, how I missed that?! Will do immediately

**Comment by mseri** [comment]
With your fix it seems that it is all good. I am merging since multiple people are asking for the new version

**Pull request merged**